jtune.py

```diff
@@ -1777,6 +1780,20 @@ def get_jmap_data(pid=None, procdetails=None):
 # Main
 user = getpass.getuser()
 
+
```

Comment:
This function needs to be above the # Main comment next to get_jmap_data. There should also only be one blank line, not 4. Rename the function to 'get_gc_log_file', and put all logic in it (including non rotated log file conditions). I'd probably recommend just having 'proc_details' as the only argument, and use the keys 'gc_log_path', and 'gc_file_rotation' available. Please have this function called at the same location where gc_log_file is assigned.